examples/siro_sandbox/sandbox_app.py

```diff
@@ -364,424 +308,10 @@ def _end_episode(self, do_reset=False):
         if do_reset and self._next_episode_exists():
             self._reset_environment()
 
-    @property
-    def _env_task_complete(self):
-        return (
-            self._end_on_success and self._metrics[self._success_measure_name]
-        )
-
-    @property
-    def lookat_offset_yaw(self):
-        return self._to_zero_2pi_range(self._lookat_offset_yaw)
-
-    @property
-    def lookat_offset_pitch(self):
-        return self._lookat_offset_pitch
-
-    def set_debug_line_render(self, debug_line_render):
-        self._debug_line_render = debug_line_render
-        self._debug_line_render.set_line_width(3)
-
-    def set_text_drawer(self, text_drawer):
-        self._text_drawer = text_drawer
-
-    def is_free_camera_mode(self):
-        return self.ctrl_helper.get_gui_controlled_agent_index() is None
-
     # trying to get around mypy complaints about missing sim attributes
     def get_sim(self) -> Any:
         return self.habitat_env.task._sim
 
-    def _draw_nav_hint_from_agent(self, end_pos, end_radius, color):
```

Comment:
The big chunks of code removed from `sandbox_app.py` have mostly been moved to `app_state_rearrange.py`, `gui_navigation_helper.py`, or other helpers.